date_guesser/constants.py

```diff
@@ -14,4 +13,32 @@ class Accuracy(IntEnum):
 
 NO_METHOD = 'Did not find anything'
 
-Guess = namedtuple('Guess', ('date', 'accuracy', 'method'))
+
+class Guess(object):
+    """Date guessing result for the provided URL and its HTML contents."""
+
+    __slots__ = [
+        '__date',
+        '__accuracy',
+        '__method',
+    ]
+
+    def __init__(self, date, accuracy = None, method = None):
```

Comment:
could you remove default arguments? the object is meant to be instantiated internally, so I would rather it be finicky.